stories/Badge.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 import { storiesOf } from "@storybook/react";
-import Badge from "../components/content/Badge";
+import Badge from "../components/base/Badge";
```

Comment:
I know we talked about this haha, but I ended up changing these to "base" components since I don't foresee these being called directly within our content. I will update the README, but kind of what I'm thinking for the component types are:

- `base` - components that are references by either `content` or `layout` components.
- `content` - components that are directly referenced within our `.mdx` pages.
- `layout` - components that make up the layout

Reply:
`base` makes sense for these smaller (almost)individual components.!